generate/src/contract/common.rs

```diff
@@ -136,3 +161,40 @@ pub(crate) fn expand(cx: &Context) -> TokenStream {
         }
     }
 }
+
```

Comment:
Note that this code was moved from `generate/src/contract/deployment.rs` to here.